### DISCUSSION TITLE
Tectonic keynote at 2022 TeX Conference

### DISCUSSION THREAD

**Discussion by jfine2358** [tectonic-typesetting/tectonic#919] [Ideas]
Peter Williams' keynote at the 2022 TeX conference is now available [on YouTube](https://youtu.be/xdgjGIEvZLk?t=821).

I learnt a lot from it, and I hope you do also. I'll be posting my comments on the talk to this discussion thread.

By the way, the URL goes to the conference live stream. In time a separate video just for the talk will be available.

**Comment by pkgw** [comment]
Thanks for posting this! My slides are now online [here](https://tug.org/tug2022/assets/html/Peter_K_G_Williams-TUG2022-slides/), including a copy of my demo of an HTML version of *XeTeX: The Program*.

**Comment by Neved4** [comment]
It seems that video got locked private but they made a separate video of the talk [here](https://youtu.be/n2htt360liY).
